### PR TITLE
[TASK] Update hints of Extension Builder

### PR DESCRIPTION
**Pull request by alexander-nitsche**
See TYPO3-Documentation/T3DocTeam#174 for details.

### CODE REVIEW

File: Documentation/ApiOverview/Context/Index.rst [hunk 1]
```diff
@@ -88,8 +88,6 @@ Property                Call
 
 **Overlay types:**
 
-:ref:`legacy settings sys_language_overlay documented in TypoScript Reference <t3tsref:setup-config-sys-language-overlay>`.
```
Comment (linawolf) [inline Documentation/ApiOverview/Context/Index.rst:91]: why did you remove this?

File: Documentation/Settings.cfg [hunk 2]
```diff
@@ -21,20 +21,20 @@ project_discussions  =
 project_home         =
 project_issues       = https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues
 project_repository   = https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi
-use_opensearch       = https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/
+use_opensearch       = https://docs.typo3.org/m/typo3/reference-coreapi/10.4/en-us/
```
Comment (linawolf) [inline Documentation/Settings.cfg:24]: These changes - though nessesary - should be made in a separate PR

**Review by linawolf — CHANGES_REQUESTED** [review]

File: Documentation/ApiOverview/Context/Index.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/ApiOverview/Context/Index.rst:91]: This was a dead link.

File: Documentation/Settings.cfg (hunk 2 above)
Comment (alexander-nitsche) [reply Documentation/Settings.cfg:24]: A separate commit should be enough, not?
